x-pack/plugins/apm/public/components/app/APMIndicesPermission/index.test.tsx

```diff
@@ -17,28 +17,31 @@ import { MockApmPluginContextWrapper } from '../../../context/ApmPluginContext/M
 
 describe('APMIndicesPermission', () => {
   it('returns empty component when api status is loading', () => {
-    spyOn(hooks, 'useFetcher').and.returnValue({
```

Comment:
it was needed for https://www.npmjs.com/package/jest-circus as it's not `jasmine` based engine, so we cannot use native jasmine anymore